tests/spellbook.py

```diff
@@ -0,0 +1,10 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from pathlib import Path
+
+
+TESTS_DIR = Path(__file__).parent
+DATA_DIR = TESTS_DIR / "data"
+PRIVATE_KEY = DATA_DIR / "private-key"
+TESTING_DB = "sqlite:////tmp/access_tokens.db"
```

Comment:
I'd say a better place would be the project root, instead of /tmp

Reply:
thanks, changed